sdk/test-utils/recorder/package.json

```diff
@@ -65,40 +58,72 @@
   "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/test-utils/recorder/",
   "sideEffects": false,
   "dependencies": {
+    "@azure/core-client": "^1.0.0",
     "@azure/core-rest-pipeline": "^1.13.0",
     "@azure/core-auth": "^1.3.2",
     "@azure/core-util": "^1.6.1",
     "@azure/logger": "^1.0.0"
   },
   "devDependencies": {
-    "@azure/core-client": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@types/express": "^4.16.0",
     "@types/fs-extra": "^8.0.0",
     "@types/chai": "^4.1.6",
-    "@types/mocha": "^10.0.0",
     "@types/node": "^18.0.0",
     "c8": "^8.0.0",
     "chai": "^4.2.0",
```

Comment:
We should be able to get rid of `chai` as `vitest` has this built in.

Reply:
updated